drivers/sensor/bh1749/bh1749.c

```diff
@@ -306,6 +306,6 @@ static const struct sensor_driver_api bh1749_driver_api = {
 
 static struct bh1749_data bh1749_data;
 
-DEVICE_DEFINE(bh1749, DT_ROHM_BH1749_0_LABEL, bh1749_init,
-	      device_pm_control_nop, &bh1749_data, NULL,
+DEVICE_DEFINE(bh1749, DT_NORDIC_NRF_I2C_5000A000_ROHM_BH1749_38_LABEL,
```

Comment:
> We can merge this as a hotfix, but I would like @anangl to verify this is the correct permanent fix.

This label contains the base address of the TWIM peripheral and the slave address of the sensor, so it's definitely not a proper one to be used in the code that is supposed to be generic (as it limits the usability of this driver to the secure domain on nRF9160 only, requires the sensor's slave address to be configured to 0x38, and forces the use of TWIM0 in communication with the sensor). `DT_INST_0_ROHM_BH1749_LABEL` should be used instead.

Reply:
Addressed in #1116.